test/common/fit_common.py

```diff
@@ -275,13 +275,13 @@ def mkargs(in_args=None):
     arg_parser.add_argument("-group", default="all",
                             help="test group to execute: 'smoke', 'regression', 'extended', default: 'all'")
     arg_parser.add_argument("-stack", default="vagrant",
-                            help="stack label (test bed), overrides -ora")
-    arg_parser.add_argument("-ora", default="localhost",
-                            help="OnRack/RackHD appliance IP address or hostname, default: localhost")
-    arg_parser.add_argument("-version", default="onrack-devel",
-                            help="OnRack package install version, example:onrack-release-0.3.0, default: onrack-devel")
+                            help="stack label (test bed), overrides -rackhd_host")
+    arg_parser.add_argument("-rackhd_host", default="localhost",
+                            help="RackHD appliance IP address or hostname, default: localhost")
+    arg_parser.add_argument("-version", default="None",
+                            help="RackHD install version, not yet implemented")
```

Comment:
Can we just remove this option for now and add it when/if it gets implemented?  It should be pretty easy to add back in and would avoid confusion in the meantime.